### ISSUE TITLE
The EK Android app doesn't open external links in a browser anymore

### ISSUE THREAD

**Issue by manuq** [endlessm/kolibri-installer-android#133]
Reference: https://phabricator.endlessm.com/T34665#977227

Seen in the new Feedback button which should open: https://endlessos.org/key-feedback
Also seen in the old Support link inside the About modal which should open: https://support.endlessos.org/en/endless-key

Both links have the `target="_blank"` that was previously opening the Support link in an external browser. This is probably a regression in the Android webview.

**Comment by manuq** [comment]
@dylanmccall it seems that this regressed in fa2ca1c212a86811631e2196bdbdb03375b36089 . There are references to a "tryOpenExternalLink" method that doesn't exist https://github.com/search?q=tryOpenExternalLink&type=code

**Comment by dylanmccall** [comment]
Oh strange. That function is inside `mActivity`, which _should_ be (or at least was, at some point) the [`PythonActivity` defined in _endlessm/python-for-android_](https://github.com/endlessm/python-for-android/tree/2022.09.04-endless/pythonforandroid/bootstraps/webview/build/src/main/java/org/kivy/android). And it looks like that function is alive and well:

https://github.com/endlessm/python-for-android/blob/2022.09.04-endless/pythonforandroid/bootstraps/webview/build/src/main/java/org/kivy/android/PythonActivity.java#L119-L132

There isn't really a reason we can't just copy the relevant code into `KolibriAndroidHelper` here (at some point we should probably put all the Java code in this project the same way upstream is doing). But I'd like to figure out why it's happening. Did we switch to a different version of python-for-android somewhere?

**Comment by manuq** [comment]
@dylanmccall ah you are right, the function is there (I wasn't looking at the corresponding p4a tag). But the external links have regressed. They are now loaded in the main webview. Here is what I see in `adb logcat`:
```
05-31 11:19:40.237  6635  6635 I EndlessKey: mMainWebView loading finished https://endlessos.typeform.com/to/Dd364YXx?typeform-source=127.0.0.1
05-31 11:19:40.237  6635  6635 I PythonActivity: displayMainWebView
```

**Comment by manuq** [comment]
Should this be considered an "Unbreak Now" since is preventing the app from being released to the production track?

**Comment by dbnicholson** [comment]
I think this is because https://github.com/kivy/python-for-android/pull/2387/commits/4045557f6e3c804f3f0412932010b775bdf168b3 was added in p4a v2021.09.05. Up until ab8051307a1a0d0086c946b1942604a9904d3845, our p4a fork was based on v2020.06.02.

So, I think the fix is to do `PythonActivity.mActivity.mOpenExternalLinksInBrowser = True` somewhere when `is_service_context()` is `False`.

**Comment by dylanmccall** [comment]
Thank you for spotting that python-for-android change, @dbnicholson! That helped a lot :star:

**Comment by erikos** [comment]
Released in Ladybird 6.2-335 (internal testing)

**Comment by vanessa-chang** [comment]
Verified Passed in Ladybird 6.2-335 (internal testing)
